lib/ufe/Global.cpp

```diff
@@ -93,14 +94,13 @@ MStatus initialize()
 	auto usdHierHandler = UsdHierarchyHandler::create();
 	auto usdTrans3dHandler = UsdTransform3dHandler::create();
 	auto usdSceneItemOpsHandler = UsdSceneItemOpsHandler::create();
-#ifdef UFE_V2_FEATURES_AVAILABLE
-	auto usdAttributesHandler = UsdAttributesHandler::create();
-	g_USDRtid = Ufe::RunTimeMgr::instance().register_(
-		kUSDRunTimeName, usdHierHandler, usdTrans3dHandler, usdSceneItemOpsHandler, usdAttributesHandler, nullptr);
-#else
+	UFE_V2(auto usdAttributesHandler = UsdAttributesHandler::create();)
+	UFE_V2(auto usdObject3dHandler = UsdObject3dHandler::create();)
 	g_USDRtid = Ufe::RunTimeMgr::instance().register_(
-		kUSDRunTimeName, usdHierHandler, usdTrans3dHandler, usdSceneItemOpsHandler);
-#endif
+		kUSDRunTimeName, usdHierHandler, usdTrans3dHandler, 
+        usdSceneItemOpsHandler
+        UFE_V2(, usdAttributesHandler, usdObject3dHandler));
```

Comment:
Register new implementation of UFE Object3d interface.  For the moment, this only has support for bounding box computation, which is used by Maya's viewFit command.